k8s/openebs-pre-release-features.yaml

```diff
@@ -93,7 +93,7 @@ metadata:
   namespace: openebs
 spec:
   meta: |
-    id: cstorpoollistnode
+    id: listnode
```

Comment:
what does this run task to? listnode or listdisksofnode?

Reply:
It will form a map of node to the disks.
The map will be formed only for the disk under investigation.